superset/assets/visualizations/country_map.js

```diff
@@ -0,0 +1,190 @@
+require('./country_map.css');
```

Comment:
We're moving towards ES6 and away from using `require`
`import './country_map.css'`